views/home.html

```diff
@@ -48,7 +48,14 @@
         <th class='version'>Version</th>
         <th>Inbound Connections</th>
         <th>Outbound Connections</th>
-        <th>Uptime</th>
+        <th class="uptime">
+          Uptime
+          <button type="button" class="tooltip-btn btn btn-default"
+            data-placement="top"
+            title="uptime of peer's longest current connection"
```

Comment:
that's why I wanted to change the column altogether. uptime of the smth is already defined as up-time, i.e. the time smth is up. we need a different word here